common.gypi

```diff
@@ -80,6 +80,7 @@
     'conditions': [
       ['target_arch=="arm64"', {
         # Disabled pending https://github.com/nodejs/node/issues/23913.
+        # Test to re-enable openssl asm for arm64.
         'openssl_no_asm%': 1,
```

Comment:
I'm confused, don't you want to change this to `0`? You just added a comment line, which won't change the build. If you change this, I'll kick off a build.

Reply:
Thanks - I missed that.